index.html

```diff
@@ -86,41 +78,169 @@ <h2 class="second-header">I am software developer</h2>
+            </div>
+            <ul class="languages">
+              <li><h3>Ruby on rails</h3></li>
+              <li><h3>css</h3></li>
+              <li><h3>JavaScript</h3></li>
+              <li><h3>html</h3></li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
           </div>
         </div>
       </div>
     </section>
     <section class="about-me">
       <div class="about-me-description">
-        <h2></h2>
-        <p class="intro-text"></p>
-        <!-- <button class="get-resume"></button> -->
+        <h2>About me</h2>
+        <p class="intro-text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="get-resume">get my Resume</button>
       </div>
 
       <div class="card-container">
         <div class="card-about">
-          <div class="img-placeholder"></div>
+          <div class="img-placeholder">
+            <img
+              src="./assets/card-images/icon-languages.svg"
+              alt="Icon image for programming languages i know"
+            />
+          </div>
+          <div class="card-details">
+            <h2>languages</h2>
+            <ul class="languages">
+              <li><h3>JavaScript</h3></li>
+              <li><h3>Ruby</h3></li>
+              <li><h3>Html</h3></li>
+              <li><h3>CSS</h3></li>
+            </ul>
+          </div>
+        </div>
+        <div class="card-about">
+          <div class="img-placeholder">
+            <img
+              src="./assets/card-images/icon-frameworks.svg"
+              alt="Icon image for programming languages i know"
+            />
+          </div>
+          <div class="card-details">
+            <h2>Frameworks</h2>
+            <ul class="languages">
+              <li><h3>Bootstrap</h3></li>
+              <li><h3>Ruby on Rails</h3></li>
+              <li><h3>Rspec</h3></li>
+              <li><h3>Capybara</h3></li>
+              <li><h3>Selenium</h3></li>
+            </ul>
+          </div>
+        </div>
+        <div class="card-about">
+          <div class="img-placeholder">
+            <img
+              src="./assets/card-images/icon-skills.svg"
+              alt="Icon image for programming languages i know"
+            />
+          </div>
           <div class="card-details">
-            <h2 class="card-description"></h2>
-            <ul class="languages"></ul>
+            <h2>Skills</h2>
+            <ul class="languages">
+              <li><h3>Codekit</h3></li>
+              <li><h3>Github</h3></li>
+              <li><h3>Codepen</h3></li>
+              <li><h3>Gitlab</h3></li>
+              <li><h3>Terminal</h3></li>
+            </ul>
           </div>
         </div>
       </div>
 
       <div class="get-in-touch">
```

Comment:
- [ ] Please stick with the margin, styles  and font family  provided in Figma design 

- The below screenshot is at 320px screen width size

![image](https://user-images.githubusercontent.com/74744787/159045042-a5256396-e3eb-4d9b-b4e3-b8ecc77327be.png)